### PR TITLE
Update apache-project-maturity-model.md

### PR DESCRIPTION
**Pull request by cottage14**
Proposed edits to improve readability. Passive voice can obscure who does what, and people whose primary language is not English can find it hard to understand.

### CODE REVIEW

File: source/apache-way/apache-project-maturity-model.md [hunk 1]
```diff
@@ -6,54 +6,53 @@ The **Apache Project Maturity Model** provides a suggested framework for evaluat
...
+Contrary to other maturity models, we do not define staged partial compliance levels. A mature Apache project complies with all the elements of this model, and other projects are welcome to adopt the elements that suit their goals.
 
 Note that we try to avoid using the word "must" below. The model describes the state of a mature project, as opposed to a set of rules. 
 
-Projects which incubate at Apache might not fit into all the parts of this model, however a major goal of incubation is to bring the project's community closer to it.
+Projects which incubate at Apache might not fit into all the parts of this model; however a major goal of incubation is to bring the project's community closer to it.
 
-Questions and feedback about this model are welcome on the <a href="https://lists.apache.org/list.html?dev@community.apache.org">comdev mailing list</a>.
+We welcome qQuestions and feedback about this model on the <a href="https://lists.apache.org/list.html?dev@community.apache.org" target="_blank">comdev mailing list</a>.
```
Comment (adityasharma7) [inline source/apache-way/apache-project-maturity-model.md:21]:
```suggestion
We welcome Questions and feedback about this model on the <a href="https://lists.apache.org/list.html?dev@community.apache.org" target="_blank">comdev mailing list</a>.
```

**Comment by bdelacretaz** [comment]
Thank you for this! Given that this is a somewhat normative document and not just prose, I suggest making the dev@community.a.o list aware of these changes before merging them. I could drive that but I probably won't have time before early January, so if you don't mind letting this sit for a while I would appreciate it.

File: source/apache-way/apache-project-maturity-model.md [hunk 2]
```diff
@@ -103,22 +102,21 @@ formats that are expected to stay readable in the long term.
...
 <dt id="RE20">RE20</dt>
 <dd>
-Releases are approved by the project's PMC (see CS10), in order to make them an act of the Foundation. 
+The project's Project Management Committee (PMC) (see CS10) approves each software release in order to make the release an act of the Foundation. 
 </dd>
 
 <dt id="RE30">RE30</dt>
 <dd>
-Releases are signed and/or distributed along with digests that can be reliably used to validate the downloaded archives. 
+Releases are signed and/or distributed along with digests that anyone can be reliably use to validate the downloaded archives.
```
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:110]: Releases are signed, and distributed along with digests and signatures that anyone can reliably use to validate the downloaded archives.

File: source/apache-way/apache-project-maturity-model.md [hunk 3]
```diff
@@ -216,23 +212,20 @@ those contributors.
 
 <dt id="CS20">CS20</dt>
 <dd>
-Decisions are made by consensus among PMC members
+Decisions reqyuire a consensus among PMC members
```
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:215]: Decisions require consensus among PMC members

File: source/apache-way/apache-project-maturity-model.md [hunk 4]
```diff
@@ -373,17 +366,17 @@ contributions but grant the ASF a perpetual copyright license for them.
...
  
  (<a href="#fndef-9b0cf71f04bcd81dddbf6199f1c771e27566611e-9">10</a>)
  </li>
  
  <li>
  <a id="fnref-d9e7a517f046358463f038f3830fef171e69f78b"></a>
-Apache projects have a private mailing list that their PMC is expected to use 
+Each Apache project has a private mailing list that its PMC is expected to use 
 only when really needed. The private list is typically used for 
 discussions about people, for example to discuss and to vote on PMC
```
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:378]: for example to discuss and to vote on PMC and committer candidates.

**Review by clr-apache — CHANGES_REQUESTED** [review]
With these changes (typos) this PR is good to go. It does not change anything substantial but simply makes the document more readable.

File: source/apache-way/apache-project-maturity-model.md (hunk 3 above)
Comment (sebbASF) [reply source/apache-way/apache-project-maturity-model.md:215]: reqyuire?

**Comment by bdelacretaz** [comment]
I created #61 based on this PR as (I think) I couldn't update this one as it's based on a different repository.

I have committed the resulting changes, many thanks to @cottage14 and all who provided input here!

The resulting commit is https://github.com/apache/comdev-site/commit/bed7a048a23ced862b654804f2c50d7dba56e9a0

This PR can now be closed, if more changes are needed we can create more specific ones. I don't seem to be able to close it, @cottage14 can you do that if you agree?

**Comment by swapnilmmane** [comment]
Thank you @bdelacretaz!

**Pull request merged**